index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Hello</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <h1>Hello Microverse!</h1>
+</body>

```

Comment:
As part of the requirement, its better to put your "< h1 >" tag in a "< header >" tag for your code to be more semantic and for SEO